aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -451,7 +472,7 @@ module aptos_framework::stake {
     }
 
     /// Allows an owner to change the delegated voter of the stake pool.
-    public entry fun set_delegated_voter_with_cap(
```

Comment:
Unnecessary entry since this can only called in scripts or by other modules anyway due to the required owner cap arg.